src/Library/demos/Popovers/main.blp

```diff
@@ -0,0 +1,91 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Popovers";
+  description: _("Display content in a container above all other content on the window");
+  Box {
+    orientation: vertical;
+    halign: center;
+
+
+    Box {
+      margin-bottom: 80;
+      margin-top: 40;
+
+      MenuButton popover_button {
+      name: 'popover_button';
+      popover: plain_popover;
+      label: "plain popover";
+      primary: true;
+      margin-end: 40;
+      }
+
+      MenuButton popover_menu_button {
+        name: "popover_menu_button";
+        label: "Popover menu";
+        margin-end: 40;
+        popover: popover_menu;
+      }
+    }
+
+    Label sub_heading {
+      label: "API References";
+      styles ["title-4"]
+    }
+
+    LinkButton {
+      label: "Popover";
+      uri: "https://docs.gtk.org/gtk4/class.Popover.html";
+    }
+
+    LinkButton {
+      label: "Popover menu";
+      uri: "https://docs.gtk.org/gtk4/class.PopoverMenu.html";
+    }
+
+    LinkButton {
+      label: "Popover menu bar";
+      uri: "https://docs.gtk.org/gtk4/class.PopoverMenuBar.html";
+    }
+
+    LinkButton {
+      margin-top: 20;
+      label: "Human Interface Guidelines";
+      uri: "https://developer.gnome.org/hig/patterns/containers/popovers.html";
+    }
+  }
+}
+
+Popover plain_popover {
+  has-arrow : true;
+  name: 'Plain popover';
+  child: Box plain-popover-box {
+    name: 'plain-popover-box';
+    Label{
+      label: 'Plain popover';
+          }
+        };
+}
+
+PopoverMenu popover_menu{
+  name: 'Popover menu';
+  menu-model: menu_app;
+}
+
+menu menu menu_app {
```

Comment:
```suggestion
menu menu_app {
```

Reply:
This is not resolved and prevents Workbench from building

> error: Unexpected tokens
at ../src/./Library/demos/Popovers/main.blp line 76 column 1:
  76 |menu menu menu_app {

Please make sure you resolved all previous comments and re-request review when it's ready

![image](https://user-images.githubusercontent.com/19673/229312584-dace1e72-9e3a-4bac-bbfb-a0072705355b.png)

Additionally, please make sure that you test your changes before submitting for review.